keras_cv/utils/rle_utils.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
```

Comment:
Must be included with a license.

Example: https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/preprocessing/aug_mix.py#L1-#L13